packages/syft/src/syft/core/tensor/ancestors.py

```diff
@@ -370,40 +379,139 @@ def copy(self) -> PhiTensorAncestor:
-        print(
-            "You can upload this Tensor to a domain node by calling domain_client.load_dataset \
-            and passing in this tensor as an asset."
+        """[DEPRECATED] This method will annotate your Tensor with metadata (an upper bound
+        and lower bound on the data, as well as the people whose data is in the dataset),
+        and thus enable Differential Privacy protection.
+
+        Note: Deprecated in 0.7.0
+        `.private` method will be removed in 0.8.0, it is replaced by `annotate_with_dp_metadata`.
+        """
+
+        _deprec_message = (
+            "This method is deprecated in v0.7.0 and will be removed in future version updates. "
+            "It is replaced with `annotate_with_dp_metadata` to provide a user-friendly experience. "
+            "One can call `help(syft.Tensor.annotate_with_dp_metadata)` to learn more about its use."
         )
-        return self.private(
-            min_val=min_val,
-            max_val=max_val,
+
+        warn(_deprec_message, DeprecationWarning, stacklevel=2)
+
+        return self.annotate_with_dp_metadata(
+            lower_bound=min_val,
+            upper_bound=max_val,
             data_subjects=data_subjects,
             skip_blocking_checks=skip_blocking_checks,
         )
 
-    def private(
+    def annotated_with_dp_metadata(
         self,
         min_val: ArrayLike,
         max_val: ArrayLike,
         data_subjects: Optional[Any] = None,
         skip_blocking_checks: bool = False,
     ) -> PhiTensorAncestor:
+        """[DEPRECATED] This method will annotate your Tensor with metadata (an upper bound
+        and lower bound on the data, as well as the people whose data is in the dataset),
+        and thus enable Differential Privacy protection.
+
+        Note: Deprecated in 0.7.0
+        `.annotated_with_dp_metadata` method will be removed in 0.8.0, it is renamed to `annotate_with_dp_metadata`.
+        """
+
+        _deprec_message = (
+            "This method is deprecated in v0.7.0 and will be removed in future version. "
+            "It is renamed to `annotate_with_dp_metadata` with function arguments `min_val` and `max_val` "
+            "renamed to `lower_bound` and `upper_bound` respectively. This has been done to simplify the definition "
+            "of the function in use. "
+            "One can call `help(syft.Tensor.annotate_with_dp_metadata)` to learn more about its use."
+        )
+
+        warn(_deprec_message, DeprecationWarning, stacklevel=2)
+
+        return self.annotate_with_dp_metadata(
+            lower_bound=min_val,
+            upper_bound=max_val,
+            data_subjects=data_subjects,
+            skip_blocking_checks=skip_blocking_checks,
+        )
+
+    def annotate_with_dp_metadata(
+        self,
+        lower_bound: ArrayLike,
+        upper_bound: ArrayLike,
+        data_subjects: Optional[Any] = None,
+        skip_blocking_checks: bool = False,
+    ) -> PhiTensorAncestor:
```

Comment:
This might return a GammaTensor depending on the data_subjects passed in so do we need to change the return type here? Same with `private`, `_private` and `annotated_with_dp_metadata`.

Reply:
Yes it should. good catch @kiendang . Will push a fix for it.

Reply:
@kiendang on checking further, the return type is actually correct. Cause we return `self`, so its always a Tensor being returned (basically a ChainedTensor). Although, its child value changes to PhiTensor or GammaTensor depending on the no of data subjects.

Reply:
I see 👌